pkg/k8shandler/dashboards.go

```diff
@@ -0,0 +1,43 @@
+package k8shandler
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/openshift/elasticsearch-operator/pkg/utils"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	defaultElasticDashboardFile = "/etc/elasticsearch-operator/files/dashboards/logging-dashboard-elasticsearch.json"
+)
+
+// CreateOrUpdateDashboards creates/updates the cluster logging dashboard ConfigMap
+func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateDashboards() error {
+	fp := utils.LookupEnvWithDefault("ES_DASHBOARD_FILE", defaultElasticDashboardFile)
+	b, err := ioutil.ReadFile(fp)
```

Comment:
https://github.com/openshift/elasticsearch-operator/blob/master/pkg/utils/utils.go#L218

Reply:
Why does this wrapper swallow the error and return an empty string? Seems very anti-go to me :thinking:

Reply:
the way it was being used before was probably an anti-pattern where we were using this function call to populate entries in a map..

Reply:
up to you to call the function, otherwise we can address how it's handling errors more in https://issues.redhat.com/browse/LOG-868